Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get update && apt-get install -y \
     python3 \
     python3-distutils \
     texinfo \
-   && rm -rf /var/lib/apt/lists/*
+   && apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
As far as I know the `rm -rf /var/lib/apt/lists/*` should be enough. It seems to be common practice